src/screens/SignInV2/elements/SlideShowItem.js

```diff
@@ -19,7 +19,6 @@ export const SlideShowItem = ({
   isLogin,
   title,
   text,
-  textSvg,
   onNextSlide = () => {},
   onPressContainer = () => {}
 }) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `textSvg` prop has been removed from the component's props. Ensure that this prop is not being used elsewhere in the component or by any parent components that may be passing it down. If it is, you'll need to refactor those instances as well.

```diff
-  textSvg,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->